### ISSUE TITLE
Wrong "Edit in github" url with Permalinks.

### ISSUE THREAD

**Issue by mathieutu** [vuejs/vuepress#1425]
- [x] I confirm that this is an issue rather than a question.

#### Version
1.0.0-alpha.43

#### What is expected?
When setting a permalink configuration and clicking on the "edit on github" link, the link should target the right place of the page in the repo.
 
#### What is actually happening?
The link is wrong, and target the permalink path.

#### Other relevant information

I've also the blog plugin.

**Comment by YunYouJun** [comment]
Can you provide more config information?

This is my config, and it works.

```javascript
module.exports = {
  themeConfig: {
    repo: 'YunYouJun/software-engineering-note',
    docsDir: 'docs',
    editLinks: true,
    editLinkText: 'Edit in GitHub'
  }
}
```

**Comment by Shigma** [comment]
You should provide more details. What is your `themeConfig` and what is this "wrong link"?

**Comment by mathieutu** [comment]
my concerned config is:
```js
permalink: ':year/:slug',
themeConfig: {
    repo: 'mathieutu/blog',
    repoLabel: 'GitHub',
    editLinks: true,
}
```
And the problem is that in the "edit link" the permalink path is used instead of the real file path: 
<img width="592" alt="Screen Shot 2019-03-11 at 13 06 37" src="https://user-images.githubusercontent.com/11351322/54122910-a9246680-43fe-11e9-8fac-120d8118cc54.png">

If I set `permalink: false` the url is ok: 
<img width="492" alt="Screen Shot 2019-03-11 at 13 10 35" src="https://user-images.githubusercontent.com/11351322/54123090-1a641980-43ff-11e9-9ce7-1e3156d2aeca.png">

**Comment by Shigma** [comment]
Aha! This issue will be fixed by #1419 😄

**Comment by mathieutu** [comment]
ok thanks!

**Comment by ulivz** [comment]
Fixed at #1419
